presto-main/src/main/java/com/facebook/presto/spiller/LocalTempStorage.java

```diff
@@ -179,6 +208,14 @@ public Path getFilePath()
         {
             return filePath;
         }
+
+        @Override
+        public String toString()
+        {
+            return "LocalTempStorageHandle{" +
```

Comment:
nit: maybe simply `return filePath.toString()` for better readability when debugging and logging?